modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/layout.js

```diff
@@ -456,6 +456,17 @@ AUI.add(
 						}
 					);
 
+					Layout.layoutHandler.delegate.dd.plug(
```

Comment:
Hey @gregory-bretall and @jonathanmccann, if you look at https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/layout.js#L379-L387 it appears that this plugin should already be added to the drag and drop delegate. It appears that the "real" bug is trying to figure out why that isn't being attached properly. I sent this over as a quick workaround for Dave.  

If this fix is kept, I think it would be better to use `Layout.getLayoutHandler().delegate.dd.plug(`. That is how it is referenced in the rest of the file. Thanks guys! :)